metriks/metriks.go

```diff
@@ -0,0 +1,124 @@
+package metriks
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/armon/go-metrics"
+	"github.com/armon/go-metrics/datadog"
+)
+
+const timerGranularity = time.Millisecond
+
+// Init will initialize the internal metrics system and build a Datadog statsd sink
+func Init(serviceName string, conf Config) error {
+	sink, err := datadog.NewDogStatsdSink(statsdAddr(conf), conf.Name)
```

Comment:
I don't think Datadog should be the default (and the only one) sink. We may want to initialize an inmemory sink (for testing) or a discard all for environments where we don't care about metrics (development).